src/api/bucket_api.js

```diff
@@ -1373,7 +1373,8 @@ module.exports = {
             required: ['resource'],
             properties: {
                 resource: { type: 'string' },
-                path: { type: 'string' }
+                path: { type: 'string' },
+                nsfs_md5_enabled: { type: 'boolean' }
```

Comment:
Okay, sounds good I will update the flag name.

Reply:
@guymguym do we want the same defaults for nsfs\noobaa buckets?

Reply:
Good question.
Today the default for backingstore buckets is true.
But the default for nsfs buckets is false.
We could add this option to account, and then based on the account that creates the bucket we will set the bucket option?
WDYT we should do?

Reply:
@guymguym  We do plan to also add an option to the account, but it would be optional, so we still need a default behavior. 

Several ways we can resolve it:
* We can decide to make the `calc_md5_etag` for one of those (bucket or account) mandatory, but it feels a bit too advanced to have as a mandatory field.
* Change the default for new "chunked buckets" (trying to push your term for it 😜) to false. For existing buckets, we can set `calc_md5_etag=true` on upgrades. This behavior might be a bit confusing.
* Have different defaults for different types

Reply:
Also, maybe the name for the flag should be `force_md5_etag`, since for some cases (S3 NS for example) we will not disable the md5 etag